plugins/scaffolder/src/components/fields/EntityPicker/schema.ts

```diff
@@ -13,72 +13,74 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { z } from 'zod';
-import { makeFieldSchemaFromZod } from '../utils';
+import { z as zod } from 'zod';
+import { makeFieldSchema } from '@backstage/plugin-scaffolder-react';
 
 /**
  * @public
  */
-export const entityQueryFilterExpressionSchema = z.record(
-  z
+export const entityQueryFilterExpressionSchema = zod.record(
```

Comment:
Hmm, I wonder if we wanna deprecate this export and have it be `(z: typeof Zod) =>` for support with the new config formats. So that `zod` is injected instead of provided?

Reply:
Fine with me; I thought about defining a new export as you suggest and deprecating/refactoring the old export to call the new one. I just wasn't sure how the export might be being used.

Reply:
I've done essentially what I stated above; I note that even though the `entityQueryFilterExpressionSchema` export is marked as `@public` it doesn't seem to actually appear in the API reports. I may push a subsequent commit to remove the `@public` annotation and test the theory that doing so won't break the build.

Reply:
Removed the original export as discussed in SIG 5/2; exported the replacement function without (misleading) TSDoc for consumption by owned/owner picker fields.